src/composables/useFlawModel.ts

```diff
@@ -194,7 +194,7 @@ export function blankFlaw(): ZodFlawType {
       workflow: '',
     },
     components: [],
-    unembargo_dt: '',
+    unembargo_dt: null,
```

Comment:
I've changed this default value to prevent the error message `Invalid date` 
![image](https://github.com/RedHatProductSecurity/osim/assets/4268580/97c3b539-39a4-49c7-8f92-24b07fe355e6)
and instead show `No date selected`
![image](https://github.com/RedHatProductSecurity/osim/assets/4268580/47b2d7d1-70cc-4f57-8fe7-4b4552ae0b0e)